geopy/geocoders/osm.py

```diff
@@ -248,6 +249,58 @@ def reverse(
             self._call_geocoder(url, timeout=timeout), exactly_one
         )
 
+    def lookup(
+            self,
+            osm_ids,
+            timeout=None,
+            addressdetails=False,
+            language=False
+    ):
+        """
+        Returns address of one or multiple OSM objects.
+
+        :param osm_ids: An individual or a list of osm_ids to lookup
+
+        :param int timeout: Time, in seconds, to wait for the geocoding service
+            to respond before raising a :class:`geopy.exc.GeocoderTimedOut`
+            exception. Set this only if you wish to override, on this call
+            only, the value set during the geocoder's initialization.
+
+            .. versionadded:: 0.97
+
+        :param bool addressdetails: Include a breakdown of the address into
+            elements
+
+        :param string language: Preferred language in which to return results.
+            Either uses standard
+            `RFC2616 <http://www.ietf.org/rfc/rfc2616.txt>`_
+            accept-language string or a simple comma-separated
+            list of language codes.
+
+        """
+        params = {'format': 'json'}
+        if addressdetails:
+            params['addressdetails'] = 1
+
+        if language:
+            params['language'] = language
+
+        if osm_ids:
+            if isinstance(osm_ids, string_compare):
```

Comment:
since `None` is not of type `string_compare`, `list`, nor `set` you could theoretically get rid of the top level `if osm_ids`